client/src/interface/AuthApiData.ts

```diff
@@ -4,6 +4,7 @@ export interface AuthApiDataSuccess {
   message: string;
   user: User;
   token: string;
+  key: string;
```

Comment:
I would say modifying the `AuthApiData` related interfaces is not the most appropriate way of doing this. Maybe we can create an interface of its own?